tools/ARIAtools/extractProduct.py

```diff
@@ -407,11 +453,42 @@ def export_products(full_product_dict, bbox_file, prods_TOTbbox, layers, dem=Non
                     outFileUnw=os.path.join(outDir,'unwrappedPhase',product_dict[1][i[0]][0])
                     outFileConnComp=os.path.join(outDir,'connectedComponents',product_dict[1][i[0]][0])
 
+                    # change outputFormat if resampling is specified
+                    if arrshape is not None and outputFormat=='VRT':
+                        outputFormat='ENVI'
+
                     # calling the stitching methods
+                    print('Stitching ({} method)'.format(stitchMethodType))
                     if stitchMethodType == 'overlap':
-                        product_stitch_overlap(unw_files,conn_files,prod_bbox_files,bounds,prods_TOTbbox, outFileUnw=outFileUnw,outFileConnComp= outFileConnComp,mask=mask,outputFormat = outputFormat,verbose=verbose)
+                        product_stitch_overlap(unw_files,conn_files,prod_bbox_files,bounds,prods_TOTbbox, outFileUnw=outFileUnw,outFileConnComp=outFileConnComp,outputFormat = outputFormat,verbose=verbose)
                     elif stitchMethodType == '2stage':
-                        product_stitch_2stage(unw_files,conn_files,bounds,prods_TOTbbox,outFileUnw=outFileUnw,outFileConnComp= outFileConnComp,mask=mask,outputFormat = outputFormat,verbose=verbose)
+                        product_stitch_2stage(unw_files,conn_files,bounds,prods_TOTbbox,outFileUnw=outFileUnw,outFileConnComp= outFileConnComp,outputFormat = outputFormat,verbose=verbose)
```

Comment:
@rzinke @sssangha I noticed the mask option has been removed here from the product_stitch_2stage. Did not see an edit to the function itself. Double check if this was not done in error.

Reply:
Oops. If this was my doing it was certainly not done on purpose.